pkg/cmd/run.go

```diff
@@ -570,6 +628,10 @@ func init() {
 	runCmd.Flags().Bool("fallback-only", false, "read all secrets directly from the fallback file, without contacting Doppler. secrets will not be updated. (implies --fallback-readonly)")
 	runCmd.Flags().Bool("no-exit-on-write-failure", false, "do not exit if unable to write the fallback file")
 	runCmd.Flags().Bool("forward-signals", forwardSignals, "forward signals to the child process (defaults to false when STDOUT is a TTY)")
+	// secrets mount flags
+	runCmd.Flags().String("mount", "", "write secrets to an ephemeral file, accessible at DOPPLER_CLI_SECRETS_PATH. when enabled, secrets are NOT injected into the environment")
+	runCmd.Flags().String("mount-format", "json", "file format to use. one of [json, env]")
+	runCmd.Flags().Int("mount-max-reads", 0, "maximum number of times the mounted secrets file can be read (0 for unlimited)")
```

Comment:
```suggestion
	runCmd.Flags().Int("mount-max-reads", 0, "number of read requests permitted (0 for unlimited)")
```

Reply:
Do you think the new language makes it clear what this feature does? I wonder if it's confusing from the perspective of a user who knows nothing about named pipes.

Reply:
I think the new language is clearer and more concise and the usage of named pipes is an implementation detail that doesn't benefit the user to know for general usage.